Source/JavaScriptCore/wasm/wasm.json

```diff
@@ -15,6 +15,7 @@
         "externref": { "type": "varint7", "value":  -17, "b3type": "B3::Int64" },
         "ref_null":  { "type": "varint7", "value":  -20, "b3type": "B3::Int64" },
         "ref":       { "type": "varint7", "value":  -21, "b3type": "B3::Int64" },
+        "i31ref":    { "type": "varint7", "value":  -22, "b3type": "B3::Void" },
```

Comment:
I had left this as `Void` here to be consistent with, e.g., structs. In particular `TypeKind::I31ref` shouldn't show up as a type on its own (same with structs), because the parser should always convert an i31ref type to `TypeKind::Ref` (or `RefNull`) with an I31 heap type.